configuration/items.md

```diff
@@ -669,6 +670,7 @@ Player MyPlayer   { expire="1h,command=STOP" } // send STOP command after one ho
 Number MyChannel  { expire="5m,state=0" }      // update state to 0 after five minutes
 String MyMessage  { expire="3m12s,Hello" }     // update state to Hello after three minutes and 12 seconds
 Switch MySwitch   { expire="2h" }              // update state to Undefined two hours after last value
+Switch MyTrigger  { channel="openhab:speedtest:binding:trigger_test", expire="5s,command=OFF" } // update state to OFF after 5 seconds
```

Comment:
We should maybe follow the [same syntax as above](https://github.com/openhab/openhab-docs/pull/1629/files#diff-f35476973205a2c93d83050c5d645b4db05826b0aeb9eafe3fa355c2c41988d6R641) with the `autoupdate` feature.
Instead of adding another Switch, I'd suggest to hence simply add `channel="xxx", ` to the three entries above.

Reply:
Agreed, i followed the example of `Text File Linking` with real values.